docs/general/data-structures.md

```diff
@@ -0,0 +1,39 @@
+# Data structures
+
+The `data` property of a dataset can be passed in various formats. By default all data is parsed using configured scales.
```

Comment:
"By default all data is parsed using configured scales." maybe would be better said as "By default, a dataset is parsed using the associated chart type and scales." to indicate that a scale doesn't parse all data but only the datasets plotted on that scale and also that the controller does a lot of the parsing

Reply:
done

Reply:
yep!